boards/hifive1/layout.ld

```diff
@@ -0,0 +1,10 @@
+MEMORY
```

Comment:
Can you reference where these values come from? I'm trying to understand why prog is 507M.